### ISSUE TITLE
RxJava 3, reactive-streams

### ISSUE THREAD

**Issue by idontusenumbers** [JetBrains/capture-points#10]
Any chance we could get a set of capture points for RxJava3?

I've tried a few myself but I can't figure it out.

**Comment by gorrus** [comment]
To me it also seems that plugin hooks + schedule/execute annotaions approach should work better than capture points setup, have you tried that?

**Comment by idontusenumbers** [comment]
I tried the plugin hooks but they didn't seem to work.

**Comment by gorrus** [comment]
Yep, unfortunately the hook at the insertion point is executed before the actual singleSupplier execution, so when we're inside the singleSupplier execution we're already unable to get the async stack. You'll have to wait for https://youtrack.jetbrains.com/issue/IDEA-186404 and then use external annotations to setup async stack traces in any library.
